packages/common/src/content/index.ts

```diff
@@ -438,11 +438,12 @@ export const getContentTypeLabel = (
 };
 
 /**
- * returns a new content that has the specified hubId and updated identifier
+ * DEPRECATED returns a new content that has the specified hubId and updated identifier
```

Comment:
If this PR removes a package... which I expect is breaking change... can we remove these functions as well?

Reply:
it's not a breaking change to remove the package from source code - the package still exists on NPM and is unchanged. 😉

Reply:
🎉🎉🎉🎉